src/components/collection/SummaryHeader.tsx

```diff
@@ -0,0 +1,187 @@
+
+import { Favorite, Visibility } from '@mui/icons-material';
+import { Skeleton } from '@mui/lab';
+import {
+  Container,
+  Grid,
+  Typography,
+  Chip,
+  Divider,
+  Tooltip,
+} from '@mui/material';
+import dynamic from 'next/dynamic';
+import React from 'react';
+// import { useTranslation } from 'react-i18next';
+import { THUMBNAIL_SIZES } from '../../config/constants';
+import { ITEM_SUMMARY_TITLE_ID, SUMMARY_TAGS_CONTAINER_ID } from '../../config/selectors';
+
+import CardMedia from '../common/CardMediaComponent';
+import { StyledCard } from '../common/StyledCard';
+import Authorship from './Authorship';
+import Badges from './Badges';
+import SummaryActionButtons from './SummaryActionButtons';
+import Description from './SummaryDescription';
+
+const {
+  LikeButton,
+} = {
+  LikeButton: dynamic(
+    () => import('@graasp/ui').then((mod) => mod.LikeButton),
+    { ssr: false },
+  ),
+};
+
+type SummaryHeaderProps = {
+  itemId: string;
+  isLoading: boolean;
+  name: string;
+  truncatedName: string;
+  tags: Immutable.List<string> | undefined;
+  description: string;
+  creator: Immutable.Map<string, string> | null;
+  views: number;
+  likes: number;
+  isLogged: boolean;
+  extra: any;
+};
+
+const DESCRIPTION_MAX_LENGTH = 250;
+
+const SummaryHeader: React.FC<SummaryHeaderProps> = ({
+  isLogged,
+  isLoading,
+  itemId,
+  name,
+  truncatedName,
+  tags,
+  description,
+  creator,
+  views,
+  likes,
+  extra,
+  // eslint-disable-next-line arrow-body-style
+}) => {
+  // const { t } = useTranslation();
+  
+  return (
+    <Container maxWidth="lg">
+      <Grid container columnSpacing={10} alignItems="center">
+        <Grid
+          item
+          sm={12}
+          md={4}
+          mb={4}
+          alignItems="center"
+          justifyContent="center"
+        >
+          <StyledCard>
+            {isLoading ? (
+              <Skeleton variant="rectangular" width="100%">
+                <CardMedia itemId={itemId} name={name} />
+              </Skeleton>
+            ) : (
+              <CardMedia
+                itemId={itemId}
+                name={name}
+                size={THUMBNAIL_SIZES.ORIGINAL}
+              />
+            )}
+          </StyledCard>
+        </Grid>
+        <Grid item sm={12} md={8}>
+          <Grid item justifyContent='space-between' flexDirection='row' display='flex' alignItems='end'>
```

Comment:
I think it looks a bit weird that way. Here is the result:
![image](https://user-images.githubusercontent.com/100228798/233867896-d605235b-84cb-4f56-b331-f49cc85e2afa.png)
![image](https://user-images.githubusercontent.com/100228798/233867906-650b14ac-a349-4ec6-9087-672711ea9e8e.png)
And centered:
![image](https://user-images.githubusercontent.com/100228798/233868001-d9eb5060-1335-437a-8203-8252f0c2a2e0.png)
![image](https://user-images.githubusercontent.com/100228798/233868017-97f4a992-1e91-4dfc-be82-ca17e9758073.png)

I also thought about adding a spacer to fill the space but it's really looking weird:
![image](https://user-images.githubusercontent.com/100228798/233868094-c3a50afe-f33b-4b3c-bc70-a24edc227185.png)

which one should we use?

Reply:
I still think aligning to the top looks better. That way when you show more the top part does not move. Maybe you find it weird because elements are not perfectly aligned. See below:
![library_top_comments](https://user-images.githubusercontent.com/39373170/234216817-27c9dc0a-5b73-4341-9f65-2f0f4890dfd0.png)
![library_top_improved](https://user-images.githubusercontent.com/39373170/234216885-d71e9bf5-22aa-4486-bc03-56ac3933b32a.png)